chains/solana/contracts/programs/ccip-router/src/context.rs

```diff
@@ -29,8 +29,8 @@ pub fn uninitialized(v: u8) -> bool {
 }
 
 // Fixed seeds - different contexts must use different PDA seeds
-pub const DEST_CHAIN_STATE_SEED: &[u8] = b"dest_chain_state";
-pub const SOURCE_CHAIN_STATE_SEED: &[u8] = b"source_chain_state";
+pub const DEST_CHAIN_SEED: &[u8] = b"dest_chain_state";
+pub const SOURCE_CHAIN_SEED: &[u8] = b"source_chain_state";
```

Comment:
I'm removing the `state` bit of the names here as they hold config & state. I left the seeds untouched to no affect offchain code that may already be using those values though. If you think it's ok to change the seed values and just notify people, I'll happily do it.